### ISSUE TITLE
Rolldown interaction with Distill

### ISSUE THREAD

**Issue by jbixon13** [uncoast-unconf/rolldown#14]
I've been hoping for a scrollytelling implementation in R for a while so I'm immensely thankful for your work. 

I'm wondering how output: rolldown::scrollama_sidebar will interact with a website built using the Distill for R Markdown package (which has its own output using similar syntax).

I'm aware this is experimental and likely has not been considered yet, just curious on your thoughts.

**Comment by iqis** [comment]
@jbixon13 Hi Jason, thank you so much for your interest. While @yihui is the expert on the subject, from my limited understanding, `rolldown::scrollama_sidebar` is parallel to `distill::distill_articles`, both as extensions to R Markdown. 

`rolldown` aims to generate minimalist, one-pager documents, which will be less complex than a Distill article. This means in the current state, if one were to use `rolldown` on a for a R Markdown file written for Distill,  while headings, `#`, `##`, etc, will correctly define content blocks to be turned into sections, unique features provided by the `distill` package, such as table of content, citations, appendices, etc will not be available. 

However, it makes an interesting topic to explore the interoperability and convertibility among different R Markdown extensions.

**Comment by jbixon13** [comment]
I agree on the complexity and importance of making extensions interoperable (especially when considering how different tools will interact), even forgetting Distill for now and thinking of Blogdown which is much more prevalent. 

I would love to help out in any way possible but don't know what the development priorities are. I don't have much experience with package development but can help out wherever is needed as this is an important tool to me and to a lot of people looking to communicate their work.

**Comment by yihui** [comment]
Give me a few more months (I'm currently stuck in other packages). I really hope **blogdown** and **distill** can play together. I believe it is entirely possible to use Scrollama on a **blogdown** website.

**Comment by vidonne** [comment]
I second Jason thanks a lot for your work super useful scrollytelling implementation. I was wondering if there were any changes on this issue or if the way to go is with blogdown?
We are looking for a simple way to create single page storyscrolling including charts, pictures, maps, etc. that will then be incorporated in our organization website. That why we were keen in using the Distill article approach and potentially mix it with rolldown. 
Looking forward hearing from you if this has a solution somewhere. Thanks again for the work!

**Comment by yihui** [comment]
Sorry, there is no progress on this issue yet. I just "officially" added it to my todo list and will let you know once I have any news. Thanks!

**Comment by yihui** [comment]
@vidonne Not yet. Sorry...

**Comment by paragemini** [comment]
@yihui love your packages especially rolldown. Would be grateful if you can combine rolldown and distill together. 
Maybe you can name the package as rollDis. Thanks for your help!
